chaco/tools/line_inspector.py

```diff
@@ -78,7 +78,8 @@ def draw(self, gc, view_bounds=None):
         if self.is_listener:
             tmp = self._get_screen_pts()
         elif self.is_interactive:
-            tmp = self._last_position
```

Comment:
Hi, I pulled in my branch #205 as I found it helped quite a bit and it looked like it will not be implemented anytime soon. I reverted changes in latest commit.